homeassistant/components/sonarr/sensor.py

```diff
@@ -106,7 +107,7 @@ class SonarrCommandsSensor(SonarrSensor):
 
     def __init__(self, sonarr: Sonarr, entry_id: str) -> None:
         """Initialize Sonarr Commands sensor."""
-        self._commands = []
+        self._commands: list[Any] = []
```

Comment:
Please type all of the lists that store library instances correctly. Check the library for the correct types.

Reply:
Of course, done.